step-03/README.md

```diff
@@ -0,0 +1,21 @@
+# step-03
+
+This application builds on step-02 by adding retrieval augmented generation (RAG).
+
```

Comment:
Do we now have a dependency on the Tesseract library being available on the system?  If so, we'll need to update this README or the root one, with the new prereq and instructions for satisfying it (https://cwiki.apache.org/confluence/display/TIKA/TikaOCR)